build-data.stubs

```diff
@@ -24,6 +25,35 @@ use XML::XPath;
 my $xml = XML::XPath->new(filename => 'libphonenumber/resources/PhoneNumberMetadata.xml');
 my @territories = $xml->find('/phoneNumberMetadata/territories/territory')->get_nodelist();
 
+my %prefix_to_timezones;
+open(my $timezones_fh, "<:encoding(utf8)", 'libphonenumber/resources/timezones/map_data.txt')
+  or die "Opening 'map_data.txt': $!";
+while (my $line = <$timezones_fh>)  {
+  $line =~ s/^\s+|\s+$//g;
+
+  # Skip empty or comment lines
+  if (!$line || $line =~ /^#/) {
+     next;
+  }
+
+  # Assert pipe deliminated
+  if ($line !~ /\|/) {
+    die "malformatted data, expected '|''";
+  }
+
+  my ($prefix, $tz_string) = split(/\|/, $line, 2);
+  unless ($tz_string) {
+    die "missing time zones";
+  }
+
+  if ($prefix_to_timezones{$prefix}) {
+    die "duplicated prefix $prefix";
```

Comment:
`die`s should generally have a newline so that a line of error text doesn't get smooshed together on the terminal with the first line of output from the next script that runs